docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -253,484 +133,518 @@ For example, if you ran the ``easylink run`` command on your laptop, the impleme
-``tests/specifications/common/pipeline.yaml`` previously::
-
-   $ pqprint results/2025_05_06_12_04_36/result.parquet 
-           foo bar  counter  added_column_0  added_column_1  added_column_2  added_column_3  added_column_4
-   0         0   a        4             0.0             1.0             2.0             3.0               4
-   1         1   b        4             0.0             1.0             2.0             3.0               4
-   2         2   c        4             0.0             1.0             2.0             3.0               4
-   3         3   d        4             0.0             1.0             2.0             3.0               4
-   4         4   e        4             0.0             1.0             2.0             3.0               4
-   ...     ...  ..      ...             ...             ...             ...             ...             ...
-   59995  9995   a        1             0.0             0.0             0.0             0.0               4
-   59996  9996   b        1             0.0             0.0             0.0             0.0               4
-   59997  9997   c        1             0.0             0.0             0.0             0.0               4
-   59998  9998   d        1             0.0             0.0             0.0             0.0               4
-   59999  9999   e        1             0.0             0.0             0.0             0.0               4
-   [60000 rows x 8 columns]
-
-
-Implementation Configuration
-----------------------------
-Additionally, implementations can be configured in the pipeline YAML. An implementation may have some settings
-that allow it to be configured in different ways for different pipelines. These settings are defined by the 
-implementation itself, rather than the pipeline schema, so it is up to the user of the implementation to 
-understand and configure them. 
-
-These settings are configured by placing environment variables in the ``configuration`` section of the 
-``implementation`` definition in the YAML. We'll use a new pipeline YAML, 
-:download:`impl-config-pipeline.yaml <impl-config-pipeline.yaml>`, as an example::
-
-   steps:
-      step_1:
-         implementation:
-            name: step_1_python_pandas
+              name: default_updating_clusters
+      canonicalizing_and_downstream_analysis:
+        implementation:
+          name: dummy_canonicalizing_and_downstream_analysis
+
+.. raw:: html
+
+  </details>
+
+The pipeline specification follows the structure defined in the :ref:`pipeline_schema`, a very important
+part of EasyLink. The EasyLink pipeline **schema** enforces the standard patterns that linkage step implementations must 
+follow, enabling easy configuration and swapping. 
+
+It defines the steps of the record linkage pipeline, the inputs and outputs for each step, and the required formats for 
+each input or output data file. 
+
+It also describes a set of operators which are used by the schema to allow customization 
```

Comment:
Similarly, it's unclear what this "It" is referring to. If it's referring to the same thing as in the previous sentence, maybe this could just be combined into the above paragraph to make it more clear.